ocs_ci/framework/pytest_customization/reports.py

```diff
@@ -78,3 +79,64 @@ def pytest_sessionfinish(session, exitstatus):
         save_reports()
     if ocsci_config.RUN["cli_params"].get("email"):
         email_reports(session)
+
+    # creating report of test cases with total time in ascending order
+    data = globalVariables.TIMEREPORT_DICT
+    sorted_data = dict(sorted(data.items(), key=lambda item: item[1]["total"]))
+    with open("time_report.txt", "a") as f:
```

Comment:
is it stored in the root it will create a file on every local computer. time_report.txt may be good to store somewhere within logs dir, to see it in magna logs. WDYTH?

Reply:
That's a nice idea, will add it to the manga logs location, thank you.